test/integration/revocation/terraform/main.tf

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
Reminder to re-run the copywrite tool so we have the correct license (BSL) on all of these files.